rosbot_gazebo/launch/multirobot_simulation.launch.py

```diff
@@ -13,18 +13,26 @@
 # limitations under the License.
 
 from launch import LaunchDescription
```

Comment:
It is possible to merge simulation and multiple_simulation, because they are very similar.
It will be greate to have 2 API:
- `ros2 launch rosbot_gazebo simulation.launch.py namespace:=robot`
- `ros2 launch rosbot_gazebo simulation.launch.py robots:="robot1={y: 2.0}; robot2={y: 3.0};"`

In case of:
`ros2 launch rosbot_gazebo simulation.launch.py namespace:=robot robots:="robot1={y: 2.0}; robot2={y: 3.0};"`
namespace should be ignored. 

It can be done by creating robots structure if `robots` is not defined then `robots="namespace={x,y,z,roll,peach,yaw};" ;)